### ISSUE TITLE
Tooltip on Table Column header label

### ISSUE THREAD

**Issue by skochev84** [patternfly-yew/patternfly-yew-quickstart#33]
Is there a way to set a tooltip when hovering over a label of a table column? Right now I can only set a String for the label like this:

       <TableColumn<Columns> label="Id" index={Columns::Id}/>

And I want to send a VNode. Something like this:

       <TableColumn<Columns> label=html!(
           <Tooltip text="This is just an example tooltip.">
               "Id"
          </Tooltip>)
          index={Columns::Id}/>

Or any other way to add a tooltip?

**Comment by ctron** [comment]
I agree, it would be a good enhancement to replace this with e.g. `Html`. Maybe this requires some custom type (newtype of `Html`), so that it is possible to assign from a string, as well as `html!`.

**Comment by skochev84** [comment]
There are probably other places where there's a string which can be refactored to accept html.

**Comment by ctron** [comment]
Probably, but even fixing one location is good enough.

**Comment by skochev84** [comment]
I see now that for Tab component is working as above:

`                <Tabs<TabIndex> detached=true {onselect} selected={*selected} r#box=true>
                    <Tab<TabIndex> index={TabIndex::Foo} title="Foo"/>
                    <Tab<TabIndex> index={TabIndex::Bar} title="Bar"/>
                    <Tab<TabIndex> index={TabIndex::Baz} title={html!(<>{"Baz "} <Badge>{"42"}</Badge></>)}/>
                </Tabs<TabIndex>>`

The title here can be html or just a string. Maybe do it the same way for the label on the columns in the grid component and other places.

**Comment by ctron** [comment]
Yes! Maybe pull this out to make it more generic. But I think this is the API to offer to the user/developer.
